app/views/ops/_schedule_form.html.haml

```diff
@@ -54,6 +54,8 @@
                          "checkchange"                 => "",
                          "selectpicker-for-select-tag" => "")
 
+    %label
+      test
```

Comment:
Test?

Reply:
Nope, ask @Hyperkid123 for writing tests :wink: 
but removing it, don't know why it is there...